### DISCUSSION TITLE
How to filter a numeric field?

### DISCUSSION THREAD

**Discussion by jhh78** [pocketbase/pocketbase#4736] [Q&A] [answered]
greetings.

I am using a numeric field to validate the token.
This field contains the date converted to a number.

I set the filter as follows, but it does not work.
Please let me know what I'm missing.

`pb.collection("members").getFirstListItem(`authKey="${authKey} && expire < ${moment().unix()}"`);`

**Comment by ganigeorgiev** [comment, marked as answer]
I don't understand the question and the provided code sample.

On first glance, it seems that the `authKey` double quotes are not properly closed.

You can try with:

```js
pb.collection("members").getFirstListItem(`authKey = "${authKey}" && expire < ${moment().unix()}`)

// or alternatively with named filter params that will auto add quotes based on the bound variable type
pb.collection("members").getFirstListItem(pb.filter("authKey = {:authKey} && expire < {:expire}", {
    authKey: authKey,
    expire:  moment().unix(),
}))
```

If the above doesn't help, please elaborate a little more on your use case and your actual collections structure (_you can export your collection as json from the Admin UI > Settings > Export collections_).

**Reply by jhh78** [reply]
Thank you for always kind answers.

It was a simple syntax error.
have a good day.
